contracts/nft/NFTInterestFacet.sol

```diff
@@ -0,0 +1,30 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.0;
+
+// Contracts
+import { NFTFacet } from "./NFTFacet.sol";
+
+// Libraries
+import { NFTLib } from "./libraries/NFTLib.sol";
+import { NumbersLib } from "../shared/libraries/NumbersLib.sol";
+
+contract NFTInterestFacet is NFTFacet {
+    /**
+     * @notice It returns the claimable interest % for a user's staked NFTs.
+     * @param nftOwner The address of the NFT owner.
+     */
+    function getClaimableInterestPercent(address nftOwner)
+        external
+        view
+        returns (uint16 claimableInterestPercent)
+    {
+        uint256[] memory stakedNFTs = NFTLib.stakedNFTs(nftOwner);
+        uint256[] memory diamondNFTs = NFTLib.nft().getOwnedTokens(
```

Comment:
there is a slightly easier way to do this:

```
uint256 userNFTs = EnumerableSet.length(
    NFTLib.s().stakedNFTs[nftOwner]
);
uint256 diamondNFTs = NFTLib.nft().balanceOf(address(this));
```

Reply:
that way don't have to copy token IDs into memory array and then only use the `length` value.

Reply:
This is what I was asking about last week. Will add it in